snap/snapcraft.yaml

```
@@ -3,11 +3,15 @@ adopt-info: get-iplayer
 summary: CLI for downloading content from BBC iplayer
 description: |
     Command Line tool for downloading tv and radio programmes from the last 30 days from the BBC iPlayer.
+version: "3.35"
 base: core22
 grade: stable 
 confinement: strict
 assumes:
   - command-chain
+architectures:
+  - amd64
+  - arm64
```

```suggestion
architectures:
  - amd64
  - arm64
  - armhf
  - ppc64el
  - s390x
```

The snap currently has releases across all of these architectures :)

---

✅